release-notes/release-notes56.md

```diff
@@ -24,7 +24,11 @@ These release notes provide an overview of the changes for {{site.productname}}
 
 The following new features and enhancements were added for the {{site.productname}} 5.6 release.
 
-### Improvement Name
+### New `image_file_types` option for customizing the recognized `smart_paste` image file extensions
+
+The new `image_file_types` option allows different image file extensions to be recognized by the `smart_paste` image URL to embedded image functionality.
+For information on the `image_file_types` and `smart_paste` options for the PowerPaste plugin, see: [PowerPaste plugin]({{site.baseurl}}/plugins/powerpaste/).
```

Comment:
```suggestion

For information on the `image_file_types` and `smart_paste` options for the PowerPaste plugin, see: [PowerPaste plugin]({{site.baseurl}}/plugins/powerpaste/).

```
Need a blank newline between paragraphs, otherwise Markdown parser will mash them into a single paragraph.